test/Main.purs

```diff
@@ -81,3 +84,19 @@ main = launchAff do
 
   result <- getVersionFromGitTag
   assertEq (Just (Tuple "v3.0.0" (version 3 0 0 Nil Nil))) result
+
```

Comment:
I think, since the code being tested here does IO (i.e. `newVersion` will prompt using the console if given certain arguments), it would probably be better to do this as an integration test instead of a unit test. Also, I'd rather not export `newVersion`.